components/file/hdfs/hdfs_test.go

```diff
@@ -29,12 +29,16 @@ import (
 	"go.beyondstorage.io/v5/types"
 )
 
-// config is the raw json data of component's Metadata configuration
-const config = `[
+const (
+	// config is the raw json data of component's Metadata configuration
+	config = `[
 				{
 					"endpoint": "tcp:127.0.0.1:9000"
 				}
 			]`
+	endpoint9000    = "127.0.0.1:9000"
```

Comment:
How about naming it as httpEndpoint
and tcpEndpoint? I did not see this kind of naming pattern before endpoint9000 🤣

Reply:
hahaha ok

Reply:
fixed